### PR TITLE
Provides bin script that runs drush on each affiliate.

### LINKED ISSUE #3349: Create bin script to run drush commands at every affiliate
Reference #3339, #3340.

CC @aaronschachter

### PR DESCRIPTION
**Pull request by sergiitk**
#### Usage
- `vagrant ssh`
- `cd /var/www/vagrant/`
- `bin/drush-intl status --fields=uri`
  image
#### What are the relevant tickets?

Closes #3349.

### CODE REVIEW

File: bin/drush-intl [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+WEB_DIR='html'
+
+# Full webroot path
+WEB_PATH=$BASE_PATH/$WEB_DIR
+
+
+# ==============================================================================
+# Functions
+
+for SITE in botswana canada congo ghana kenya indonesia nigeria training uk
```
Comment (aaronschachter) [inline bin/drush-intl:19]: Not necessarily for this PR, but we should really have this list as a variable somewhere.  Lots of gotchas where we miss an affiliate.

**Comment by aaronschachter** [comment]
:+1: Much value added.

Can we give it a shot by disabling and uninstalling `dosomething_zendesk` ? :smile:

File: bin/drush-intl (hunk 1 above)
Comment (sergiitk) [inline bin/drush-intl:19]: Yes, def. We should discuss and find a common way to provide a source with this list. Will create ticket for it.

**Comment by sergiitk** [comment]
Yes, @aaronschachter, let's do it!

**Pull request merged**